spec/i18n_spec.rb

```diff
@@ -3,157 +3,18 @@
 require 'rails_helper'
 require 'i18n/tasks'
 
```

Comment:
It would be good to have a comment here explaining why these variables don't live in the Rspec block (i.e., performance reasons).

Some additional context:
After experimentation, RSpec League found that local variables we're the _fastest_ by an order of magnitude (~1s vs ~10s) than declaring these as instance variables in a `before :all` block.